codex-ui/src/vue/components/alert/core/useAlert.ts

```diff
@@ -0,0 +1,105 @@
+import { ref } from 'vue';
+import { createSharedComposable } from '@vueuse/core';
+import type { AlertInterface, AlertOptions, AlertType } from './types';
+import { getId } from './constant';
+
+const useStore = createSharedComposable(() => {
```

Comment:
I'd suggest to get rid of this composable and just use a ref at the `useAlerts`

```ts
const alerts = ref<Alert[]>([]);
```

Reply:
its has being moved